src/js/components/LeftPane.js

```diff
@@ -31,6 +32,7 @@ const StyledSection = styled.section`
   min-height: 24px;
   flex-basis: 0%;
   flex-shrink: 1;
+  padding-bottom: 24px;
```

Comment:
I noticed this in this PR as well. This padding is needed for each sectionContents to be able to scroll through all of it's contents properly since we are setting the height dynamically on the containing section element now and using flex instead of setting the height on the actual sectionContents.